app/assets/javascript/application.js

```diff
@@ -9,6 +9,33 @@ jQuery(function($) {
   });
 
   ////
+  // Add "select all"/"clear all" buttons to each select2 dropdown menu
+  $("select.select2").each(function() {
+    var select_id = $(this).attr("id");
+    $(this).parent()
+      .append('<a href="#" class="btn btn-default select2-select-all" data-select-id="#' + select_id  + '">Select all</a>')
```

Comment:
Yeah, I thought buttons would stand out more but I guess they could also be confused for buttons that will make you lose your place/unsaved edits. I have turned them into links which also has the good side-effect of saving space.